lib/ops/opsservice/configure.go

```diff
@@ -995,6 +995,10 @@ func (s *site) getPlanetConfigPackage(
 		args = append(args, fmt.Sprintf("--device=%v", device.Format()))
 	}
 
+	for _, taint := range profile.Taints {
+		args = append(args, fmt.Sprintf("--taint=%v=%v:%v", taint.Key, taint.Value, taint.Effect))
```

Comment:
Is planet gonna parse this flag okay? (I mean, the fact that there is "=" in the middle.)

Reply:
It seems to work when I tested it with a local build.